.goreleaser.yaml

```diff
@@ -0,0 +1,45 @@
+# Goreleaser documentation at https://goreleaser.com
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+builds:
+  -
+    id: scalingo
+    binary: scalingo
+    main: ./scalingo
+    goos:
+      - linux
+      - windows
+      - darwin
+      - freebsd
+      - openbsd
+    goarch:
+      - amd64
+      - arm64
+      - 386
+    ignore:
+      - goos: darwin
+        goarch: 386
+
+    # Custom ldflags templates.
+    # https://goreleaser.com/customization/templates/
+    ldflags:
+      - -X main.version={{.Version}}
```

Comment:
I wonder if we would prefer `Version` instead of `version`.
But I'm not sure why we are using this variable.

Reply:
This line replaces the value of the variable `version` in the package `main`. This variable does not exist. I would say this line could be removed.